docs/TESTING.md

```
@@ -0,0 +1,113 @@
+secret = fake_secret()      # 'secret-<hex>'
+```
+
+Where a test asserts equality, capture the returned value rather than comparing to
+a literal.
+
+## 2. Local integration — `tests/test_integration_fake.py`, run on every push/PR
+
+A localhost fake of the Console HTTP API **and** the provider-proxy WebSocket
+(`tests/_fake_akash.py`), so the **full CLI runs end-to-end without credentials**:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)